samples/apps/autogen-assistant/README.md

```diff
@@ -0,0 +1,87 @@
+# AutoGen Research Assistant
+
+![ARA](./docs/ara_stockprices.png)
+
+AutoGen Research Assistant is an Autogen-powered AI research assistant that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills by demonstration, and adapt in response to your interactions.
+
+Project Structure:
+
+- _autogenra/_ code for the backend classes and web api (FastAPI)
+- _frontend/_ code for the webui, built with Gatsby and Tailwind
+
+## Getting Started
+
+Install requirements:
+
+```bash
+pip install -e .
+```
+
+AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM. We recommend setting up your `OPENAI_API_KEY` environment variable first and `OAI_CONFIG_LIST` environment variable to contain a list of OpenAI model configurations.
```

Comment:
Why do we need both an OPENAI_API_KEY and an OAI_CONFIG_LIST? Isn't it an either/or?

Reply:
Thanks.
True. 
This specific app encourages the use of agent specs (which includes `llm_config`). 
For now, I will change language to the below (focus on setting OPEN_AI_API_KEY, and modifying the agent spec) and perhaps explore better support for `OAI_CONFIG_LIST` in a future PR. 

---
AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM provider. In this sample, We recommend setting up your `OPENAI_API_KEY` environment variable and then specifying the exact model names to be used in the `llm_config` that is passed to each agent specification. See the `get_default_agent_config()` method in  `utils.py` to see an example of setting up `llm_config`